frontend/src/scenes/scenes.ts

```diff
@@ -541,6 +547,7 @@ export const routes: Record<string, [Scene | string, string]> = {
     [urls.savedInsights()]: [Scene.SavedInsights, 'savedInsights'],
     [urls.webAnalytics()]: [Scene.WebAnalytics, 'webAnalytics'],
     [urls.webAnalyticsWebVitals()]: [Scene.WebAnalytics, 'webAnalyticsWebVitals'],
+    [urls.webAnalyticsPageReports()]: [Scene.WebAnalytics, 'webAnalyticsPageReports'],
```

Comment:
I'm getting that this doesn't exist, I think atm you need to add it to both `products` and `urls`

Reply:
I think I had the same issue as you did when running after merging with master. 

Urls now need to be placed on: `product folder manifests under products/*/manifest.tsx`.
after #29728